coalib/bearlib/aspects/decorators.py

```diff
@@ -49,3 +49,51 @@ def _new_func(self, *args, **kwargs):
         return _new_func
 
     return _func_decorator
+
+
+def map_ambiguous_setting_to_aspect(**aspectable_setting):
+    """
+    Map those bear settings to aspects in which there is some ambiguity.
```

Comment:
Consider "Specify aspects to use instead of the default bear settings where there is some ambiguity.

Reply:
I don't think it's relevant to add here as ambiguity is purely because of differing values (maybe either name or even data-types) in bear settings and their corresponding aspect tastes. 

Bear settings itself are not ambiguous. :)